src/main.js

```diff
@@ -624,7 +624,7 @@ const DiscoveryFeedVideoCardLayout = new Lang.Class({
 
         this.title_label.label = this.title;
         this.app_label.label = this.app_name;
-        this.duration.label = parseDuration(this.duration);
+        this.duration.label = this.duration;
```

Comment:
Should this not be `this.duration_label.label`?

Reply:
Ah you caught the bug I had been looking for ;-)